configsnap

```diff
@@ -519,7 +519,7 @@ if os.path.exists(customcollectionfile):
 for section in Config.sections():
     if Config.has_option(section, 'type'):
         if not re.match('^(file|directory|command)$', Config.get(section, 'type').lower()):
-        #if Config.get(section, 'type').lower() != "file" and Config.get(section, 'type').lower() != "command":
+            # if Config.get(section, 'type').lower() != "file" and Config.get(section, 'type').lower() != "command":
```

Comment:
Sorry Nick, can we just get rid of that old line? I shouldn't have kept it in the patch.